test/app-luatest/datetime_test.lua

```diff
@@ -0,0 +1,2101 @@
+            buf = 'P1DT1H/2024-213T17:42',
+        }, {
+            fmt = 'P1DT1H/%Y-%OT%h:%m:%s',
+            buf = 'P1DT1H/2024-213T17:42:56',
+        }, {
+            fmt = 'P1DT1H/%Y-%OT%h:%m:%.3s',
+            buf = 'P1DT1H/2024-213T17:42:56.132',
+        }, {
+            fmt = 'P1DT1H/%Y-%OT%h:%mZ',
+            buf = 'P1DT1H/2024-213T14:42Z',
+        }, {
+            fmt = 'R/%Y-%M-%D/P1Y',
+            buf = 'R/2024-07-31/P1Y',
+        }, {
+            fmt = 'R/%V-W%W-%w/P1Y',
+            buf = 'R/2024-W31-3/P1Y',
+        }, {
+            fmt = 'R/%Y-%O/P1Y',
+            buf = 'R/2024-213/P1Y',
+        }, {
+            fmt = 'R/%Y-%M-%D/%Y-%M-%D',
+            buf = 'R/2024-07-31/2024-07-31',
+        }, {
+            fmt = 'R/%Y-%M-%D/%V-W%W-%w',
+            buf = 'R/2024-07-31/2024-W31-3',
+        }, {
+            fmt = 'R/%Y-%M-%D/%Y-%O',
+            buf = 'R/2024-07-31/2024-213',
+        }, {
+            fmt = 'R/%V-W%W-%w/%Y-%M-%D',
+            buf = 'R/2024-W31-3/2024-07-31',
+        }, {
+            fmt = 'R/%V-W%W-%w/%V-W%W-%w',
+            buf = 'R/2024-W31-3/2024-W31-3',
+        }, {
+            fmt = 'R/%V-W%W-%w/%Y-%O',
+            buf = 'R/2024-W31-3/2024-213',
+        }, {
+            fmt = 'R/%Y-%O/%Y-%M-%D',
+            buf = 'R/2024-213/2024-07-31',
+        }, {
+            fmt = 'R/%Y-%O/%V-W%W-%w',
+            buf = 'R/2024-213/2024-W31-3',
+        }, {
+            fmt = 'R/%Y-%O/%Y-%O',
+            buf = 'R/2024-213/2024-213',
+        }, {
+            fmt = 'R10/%Y-%M-%D/P1Y',
+            buf = 'R10/2024-07-31/P1Y',
+        }, {
+            fmt = 'R10/%V-W%W-%w/P1Y',
+            buf = 'R10/2024-W31-3/P1Y',
+        }, {
+            fmt = 'R10/%Y-%O/P1Y',
+            buf = 'R10/2024-213/P1Y',
+        }, {
+            fmt = 'R10/%Y-%M-%D/%Y-%M-%D',
+            buf = 'R10/2024-07-31/2024-07-31',
+        }, {
+            fmt = 'R10/%Y-%M-%D/%V-W%W-%w',
+            buf = 'R10/2024-07-31/2024-W31-3',
+        }, {
+            fmt = 'R10/%Y-%M-%D/%Y-%O',
+            buf = 'R10/2024-07-31/2024-213',
+        }, {
+            fmt = 'R10/%V-W%W-%w/%Y-%M-%D',
+            buf = 'R10/2024-W31-3/2024-07-31',
+        }, {
+            fmt = 'R10/%V-W%W-%w/%V-W%W-%w',
+            buf = 'R10/2024-W31-3/2024-W31-3',
+        }, {
+            fmt = 'R10/%V-W%W-%w/%Y-%O',
+            buf = 'R10/2024-W31-3/2024-213',
+        }, {
+            fmt = 'R10/%Y-%O/%Y-%M-%D',
+            buf = 'R10/2024-213/2024-07-31',
+        }, {
+            fmt = 'R10/%Y-%O/%V-W%W-%w',
+            buf = 'R10/2024-213/2024-W31-3',
+        }, {
+            fmt = 'R10/%Y-%O/%Y-%O',
+            buf = 'R10/2024-213/2024-213',
+        },
+    }
+
+}
+
+local pg = t.group('pgroup')
+
+-- XXX: It is not possible to use parameterization by passing a
+-- table with test parameters to `t.group` because datetime format
+-- strings in test parameters contains symbol `/` that is not
```

Comment:
Fixed.

```diff
@@ -2040,7 +2044,7 @@ local pg = t.group('pgroup')
 
 -- XXX: It is not possible to use parameterization by passing a
 -- table with test parameters to `t.group` because datetime format
--- strings in test parameters contains symbol `/` that is not
+-- strings in test parameters contains the symbol `/` that is not
 -- allowed in testcases names. Source code below inserts
 -- test functions to a test group with testnames where `/` is
 -- replaced with `_`.
```